distributed/tensor_parallelism/README.md

```diff
@@ -0,0 +1,16 @@
+# PyTorch Tensor Parallelism for distributed training
+
+This example demonstrates SPMD Megatron-LM style tensor parallel by using
+PyTorch native module parallelise APIs, which include:
+
+1. High-level APIs for module-level distribution with a dummy MLP model.
+2. Model agnostic ops for `DistributedTensor`, such as `Linear` and `RELU`.
```

Comment:
what does "Model Agnostic ops" mean?

Reply:
It means these ops work anyway for DTensor in all circumstances.